init.lua

```diff
@@ -200,9 +224,10 @@ function SkyRocket:handleClick()
     if self.dragging then return true end
 
     local flags = event:getFlags()
+    local etype = event:getType()
```

Comment:
**nit** Prefer expressive and clear variable names like `eventType`. Abbreviating `event` to `e` obscures what this variable is.